packages/workflow/src/WorkflowActivationError.ts

```diff
@@ -23,5 +24,15 @@ export class WorkflowActivationError extends ExecutionBaseError {
 		super(message, { cause: error });
 		this.node = node;
 		this.message = message;
+		if (severity) this.severity = severity;
+	}
+}
+
+export class WebhookPathAlreadyTakenError extends WorkflowActivationError {
+	constructor(webhookData: { node: string }, cause?: Error) {
```

Comment:
If we only need the node name in the first arg, then we should only pass the node name there. Or perhaps we can enrich the error message with the already existing path.

Reply:
I find this ever growing inheritance chain of execution errors harder and harder to follow. Out of scope but I'd love for us to see if we can flatten it in future.

Reply:
Addressed the first comment. Regarding the second comment: yeah, we should take some time to restructure all our error classes. it could all be much simpler.